### DISCUSSION TITLE
Idee - Tipologia lezione, interfaccia voti, didattica

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#491] [Idee e proposte]
Avevo in mente alcune idee per implementare il registro elettronico.
- Una l’avevo già proposta qui https://github.com/iisgiua/giuaschool/discussions/98#discussion-3784149 
- La seconda sarebbe relativa all’interfaccia dei voti, che ora è un po’ disordinata: è possibile creare delle colonne per data, in modo che sia visibile meglio il numero delle valutazioni per alunno in una determinata data e in modo che risulti chiaro quale alunni hanno voti in più/in meno rispetto agli altri? (Come, ad esempio, la separazione che fa classeviva)
- La terza riguarda l’implementazione della sezione relativa alla condivisione dei documenti: ora sono visibili solo documenti 15 maggio, programma svolto; è possibile aggiungere una sezione per la condivisione di documenti/link con genitori e alunni?
Grazie.

**Comment by trinko** [comment]
Per quello che riguarda la gestione dei documenti, era prevista una gestione come per gli avvisi/circolari. Penso che riuscirò a implementarla l'anno prossimo. (comunque i link si possono inserire nei testi delle comunicazioni, degli argomenti lezioni ecc. e vengono riconosciuti correttamente).

Le altre le segno come richieste.
